sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/KeyVaultCredentialPolicy.java

```diff
@@ -201,7 +209,22 @@ public Mono<Boolean> authorizeRequestOnChallenge(HttpPipelineCallContext context
 
             TokenRequestContext tokenRequestContext = new TokenRequestContext()
                 .addScopes(this.challenge.getScopes())
-                .setTenantId(this.challenge.getTenantId());
+                .setTenantId(this.challenge.getTenantId())
+                .setCaeEnabled(true);
+
+            String error = challengeAttributes.get("error");
+
+            if (error != null) {
+                LOGGER.verbose(String.format("The challenge response contained an error: %s", error));
```

Comment:
nit: this is a log message, use the loggers formatting instead of String.format

```suggestion
                LOGGER.verbose("The challenge response contained an error: {}", error);
```